backend/audios.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const router = express.Router();
+const { PrismaClient } = require('@prisma/client')
+const prisma = new PrismaClient()
+
+
+//Save audio recording to DB 
+router.post("/users/:id/audios/create", async (req, res) => {
```

Comment:
It would be helpful to add typing to your function parameters.
This would allow you catch type errors before running your code.
In js types are only known at runtime and so unless you add the types you would not be able to tell if there is an error in your typing until you run your code. 
Example:

```
router.post("/users/:id/audios/create", async (req: Request, res: Response) => {
```